packages/core/src/types.ts

```diff
@@ -514,13 +514,13 @@ export interface InvokeConfig<TContext, TEvent extends EventObject> {
    */
   onDone?:
     | string
-    | SingleOrArray<TransitionConfig<TContext, DoneInvokeEvent<any>>>;
+    | SingleOrArray<string | TransitionConfig<TContext, DoneInvokeEvent<any>>>;
```

Comment:
Other places like this use `TransitionConfigOrTarget`. Could you use this to keep things consistent?

Reply:
Sure thing! I've made this change.